templates/deployment-zabbix-server.yaml

```diff
@@ -49,7 +49,11 @@ spec:
       {{- if gt (.Values.zabbixserver.replicaCount | int) 1 }}
       initContainers:
         - name: init-wait-for-database-schema
+          {{- if .Values.zabbixserver_image.tag }}
```

Comment:
Hi @scoopex!

Can you change `{{- if .Values.zabbixserver_image.tag }}` by `{{- if .Values.zabbixserver.image.tag }}`?

Reply:
Sure, done :-)